blogs/dns_tracking/dns_tracking.md

```diff
@@ -0,0 +1,78 @@
+# Network Observability Per Flow DNS tracking
+
+<p align="center">
+  <img src="dns_tracking_logo.png" alt="logo" width="25%"/>
+</p>
+
+By: Julien Pinsonneau, Mehul Modi and Mohamed S. Mahmoud
+
+In today's interconnected digital landscape, Domain Name System (DNS) tracking
```

Comment:
I'd like to see more general information about network observability -- this is an opportunity to raise more awareness on the overall solution -- if we don't promote, we are missing the opportunity.  Even if we just talk about this being part of NOO and provide links to other information.  To me, this is less about this specific feature and more about how NOO has been enhanced now that we have it.